tasks/sep/fp-recovery/002-fallback-permissioned-game/README.md

```diff
@@ -0,0 +1,54 @@
+# Deputy Guardian - Fall back to `PermissionedDisputeGame`
+
+This batch udates the `respectedGameType` to `PERMISSIONED_CANNON` in the `OptimismPortal`. This action requires all in-progress withdrawals to be re-proven against a new `PermissionedDisputeGame` that was created after this update occurs.
+
+The batch will be executed on chain ID `11155111`, and contains `1` transactions.
+
+## Tx #1: Update `respectedGameType` in the `OptimismPortal`
+
+Updates the `respectedGameType` to `PERMISSIONED_CANNON` in the `OptimismPortal`, enabling permissioned proposals and challenging.
+
+**Function Signature:** `setRespectedGameType(address,uint32)`
+
+**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`
```

Comment:
This is the `DeputyGuardianModule`. It should be the portal at `0x16Fc5058F25648194471939df75CF27A2fdC48BC`,

Reply:
Spoke offline; Should be the guardian module

Reply:
nvmind. I see we're going through the module from now on. good looks.